screens/Maps/EventMap.tsx

```diff
@@ -26,6 +26,15 @@ const EventMap = () => {
   const events = route.params.events
   const navigation = useNavigation()
 
+  const computeEventDate = (date: string) => {
+    const eventDate = new Date(date).toLocaleDateString("en-US", {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric'
+    })
+    return eventDate
+  }
+
```

Comment:
I can see this function being repeatedly used in different files. I suggest defining this function externally and importing it in each file where you use it.